client/layout/index.jsx

```diff
@@ -31,6 +31,7 @@ import { getCurrentOAuth2Client } from 'calypso/state/oauth2-clients/ui/selector
 import { getPreference } from 'calypso/state/preferences/selectors';
 import isNavUnificationEnabled from 'calypso/state/selectors/is-nav-unification-enabled';
 import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
+import isWooSiteSelector from 'calypso/state/selectors/is-woo-site';
```

Comment:
Somewhere you also need to dispatch the `requestWooAdminMenu` action to trigger the REST request. I don't see it anywhere.

Instead of using Redux and data layer for this, would you be willing to try out React Query here? The code should be much shorter than the current reducer+actions+selectors+data-layer combination.

Reply:
>Instead of using Redux and data layer for this, would you be willing to try out React Query here?

Definitely!  Thanks for the suggestion, @jsnajdr.